client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -142,6 +146,46 @@ angular
         $scope.setAction('multisite/update/hosting-multisite-update', domain);
       };
 
+      $scope.restartDomain = domain => HostingDomain.restartAttachedDomain(
+        $stateParams.productId,
+        domain.name,
+      ).then(() => {
+        Alerter.success(
+          $translate.instant('hosting_tab_DOMAINS_multisite_restart_start'),
+          $scope.alerts.main,
+        );
+        _.extend(_.findWhere($scope.domains.list.results, { name: domain.name }), { status: 'updating' });
```

Comment:
This updates the model instantly and don't need to refresh the table again. The alternative would be to re fetch the whole `attachedDomain` results

Reply:
I just saying that you should use `assign` instead of `extend` and `find` instead of `findWhere` if you can :)

Both `extend` and `findWhere` are deprecated in Lodash 4

Reply:
oh got it :) done 👍